### PR TITLE
Add teams/governance to contributing.md

### PR DESCRIPTION
**Pull request by e-lo**
This pull request implements the initial governance of the TIDES data schemas as agreed to by the BDFNow John Levin.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+- Leadership  
+- Product Management  
+- Contributors  
+- Stakeholders  
+
+### Leadership
+
+Responsible for overall direction and decisionmaking on the project including: working group composition and scopes, approval of the final specification designs for v1.0 (based on the feedback of stakeholders), and approval of the governance plan created by the governance working group.
+
+Members: John Levin, Metropolitan Transit (MN)
```
Comment (jlstpaul) [inline CONTRIBUTING.md:20]: Metro Transit (MN)

**Review by j-meelah — APPROVED** [review]

**Review by botanize — COMMENTED** [review]
This document lays out organization, but doesn't provide any information on how to contribute to the spec. 

At the very least, it seems like we should define how the outlined roles relate to actual contributions. For example: 

- Product Management team: reviews & approves pull-requests.
- Contributors: contribute through branches on this repository
- Stakeholders/Public: contribute through branches on forks of this repository

Some additional recommended items that are missing:
- Steps for creating good issues or pull requests.
- How pull-requests are reviewed and approved (is there a public voting system like GTFS? Does it just require to approvals from the PMT?)
- Links to external documentation, mailing lists, or a code of conduct.
- Community and behavioral expectations.

**Comment by jlstpaul** [comment]
I apologize if this is a breach of protocol, but I'd like to suggest some significant changes and expansion to this CONTRIBUTING.md document.  I have created a Google Doc [here](https://docs.google.com/document/d/1aWWIwlWY_BJtTrztuNrdACG0GjdwS6LV9lOEzzLqEjw/edit?usp=sharing)   Can we edit that document until it is close, and then use it to seed back into repository.

**Comment by e-lo** [comment]
@jlstpaul : that's fine, I'll close this PR.  In the future, you can just add a commit to this PR to update it instead of opening a new one.

**Pull request closed without merging**